crates/next-core/js/src/entry/server-edge-api.tsx

```diff
@@ -22,7 +22,7 @@ startHandler(async ({ request, response, query, params, path }) => {
       join(process.cwd(), ".next/server/pages", chunk)
     ),
     wasm: [],
-    env: [],
+    env: Object.keys(process.env),
```

Comment:
This was the actual fix I needed for `front`, but fixing the real env also isn't a bad thing

Reply:
Technically it should only list env vars used in code, but I guess for dev mode it doesn't matter

Reply:
that's true, maybe something we can address in the future